src/robusta/cli/utils.py

```diff
@@ -57,6 +57,26 @@ def exec_in_robusta_runner(
     return subprocess.check_call(cmd)
 
 
+def exec_in_robusta_runner_output(command, namespace) -> Optional[bytes]:
+    exec_cmd = [
```

Comment:
to avoid code duplication, I would extract the command building to one function (lines 61-73).